tests/bud.bats

```diff
@@ -3981,6 +3981,36 @@ _EOF
   run_buildah rmi myalpine
 }
 
+@test "build verify cache behaviour with --cache-ttl" {
+  mkdir -p ${TEST_SCRATCH_DIR}/bud/platform
```

Comment:
Call `_prefetch alpine` here import a copy of the base image from an offline cache rather than pulling it from the registry.